catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -331,4 +334,11 @@ private void closeNotificationBar() {
 			sendBroadcast(it);
 		}
 	}
+
+	public static int getUniqueNotificationId() {
+		if (uniqueNotificationIdCounter == null) {
+			uniqueNotificationIdCounter = new AtomicInteger(0);
+		}
```

Comment:
Hmm, we're just talking about the if block, not the whole method. Does this answer your questions?

```java
public int getUniqueNotificationId() {
	return uniqueNotificationIdCounter.incrementAndGet();
}
```